ansible/roles/ocp-workload-odo-workshop/tasks/remove_workload.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: post_workload Tasks Complete
+  debug:
+    msg: "Pre-Software checks completed successfully - Removed"
+
+- name: Remove odo workshop resources
+  shell: "oc delete \
+  services,routes,deploymentconfigs,imagestreams,secrets,configmaps,serviceaccounts,rolebindings,serviceaccounts,rolebindings,persistentvolumeclaims,pods \
+  --selector 'app={{odo_application_name}}-{{odo_project_name}},spawner=learning-portal'"
+  ignore_errors: true
+
+- name: Remove odo workshop cluster resources
+  shell: "oc delete \
+  clusterrolebindings,clusterroles \
+  --selector 'app={{odo_application_name}}-{{odo_project_name}},spawner=learning-portal'"
+  ignore_errors: true
+
+- name: Remove odo workshop project
+  shell: "oc delete project {{odo_project_name}}"
+  ignore_errors: true
```

Comment:
i would not put `ignore_errors: true` everywhere

You can check for existence of the project, if it exists, then delete.